src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -1050,7 +1050,13 @@ std::pair< bool, nuraft::cb_func::ReturnCode > RaftReplDev::handle_raft_event(nu
                 if (entry->get_val_type() != nuraft::log_val_type::app_log) { continue; }
                 if (entry->get_buf_ptr()->size() == 0) { continue; }
                 auto req = m_state_machine->localize_journal_entry_prepare(*entry);
-                if (req == nullptr) {
+                // TODO :: we need to indentify whether this log entry should be appended to log store.
```

Comment:
is #1 safe here ? as we dont have term in the rreq, how can we ensure this is not a re-written

Reply:
we have term in rreq. every rreq is identified by {originator, term , dsn}